client/src/modules/stock/lots-duplicates/lots-duplicates.js

```diff
@@ -124,6 +124,20 @@ function DuplicateLotsController(
     gridColumns.openConfigurationModal();
   }
 
+  // Call the server function to merge lots automatically
+  function autoMergeLots() {
+    vm.loading = true;
+    Lots.autoMerge()
+      .then((res) => {
+        const msg = $translate.instant('LOTS.MERGED_LOTS_AUTOMATICALLY', res);
+        load();
+        Notify.success(msg);
+        vm.loading = false;
```

Comment:
Looks like you are going to have competition here between `vm.loading = false` and the first call of `load()` where `vm.loading = true`.

Can you move the `vm.loading = false` to above the `load()` call?

-- this is a non-blocking comment since I don't think it functionality makes a difference.

Reply:
Done. Rebased.